google/cloud/spanner/CMakeLists.txt

```diff
@@ -387,8 +389,16 @@ install(EXPORT spanner-targets
 install(TARGETS spanner_client
         EXPORT spanner-targets
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+                COMPONENT google_cloud_cpp_spanner_runtime
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+                COMPONENT google_cloud_cpp_spanner_development
```

Comment:
Is this for the shared library itself? Then shouldn't it be `runtime`?

I guess the files in the `runtime` goes into the normal libxxx style packages and `development` goes to libxxx-dev.

Reply:
Yes, fixed.